app/components/Home.vue

```diff
@@ -1,30 +1,36 @@
 <template>
     <Page>
         <ActionBar>
-            <Label text="Pick Up Events" color="#18261F" />
+            <Label text="Pick Up Events" color="#ffffff" />
         </ActionBar>
-        <ScrollView>
-            <Events />
-        </ScrollView>
-    </Page>
+        <GridLayout rows="*" ref="rootLayout">
```

Comment:
Do we even need a `GridLayout`? Can we just have two tags not nested inside anything?